src/librustc_mir/borrow_check/diagnostics/explain_borrow.rs

```diff
@@ -206,6 +206,28 @@ impl BorrowExplanation {
                         ),
                     );
                 };
+
+                if let ConstraintCategory::OpaqueType = category {
+                    if let Ok(snippet) = tcx.sess.source_map().span_to_snippet(span) {
+                        let suggestable_name = if region_name.was_named() {
+                            region_name.to_string()
+                        } else {
+                            "'_".to_string()
+                        };
+
+                        err.span_suggestion(
+                            span,
+                            &format!(
+                                "you can add a constraint to the {}to make it last less than \
```

Comment:
```suggestion
                                "you can add a bound to the {}to make it last less than \
```

Reply:
I used the same text as https://github.com/rust-lang/rust/blob/master/src/librustc/infer/error_reporting/nice_region_error/static_impl_trait.rs#L57 (but I agree it's less "correct" than `bound`).
Should I change both or just this one?

Reply:
please change both